documentation/release_notes.rst

```diff
@@ -8,6 +8,85 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
+  with device policies for large data sizes.
+- Improved performance of ``set_intersection``, ``set_union``, ``set_difference``, ``set_symmetric_difference``
+  algorithms with device policies.
+- Improved performance of ``reduce_by_segment`` with device policies and executed on a GPU.
+- Improved performance of ``adjacent_difference``, ``binary_search``, ``copy``, ``copy_n``, ``fill``, ``fill_n``,
+  ``for_each``, ``for_each_n``, ``generate``, ``generate_n``, ``lower_bound``, ``move``, ``replace``, ``replace_copy``,
+  ``replace_copy_if``, ``replace_if``, ``reverse``, ``reverse_copy``, ``rotate``, ``rotate_copy``, ``shift_left``,
+  ``shift_right``, ``swap_ranges``, ``transform``, ``transform_if``, and ``upper_bound`` with device policies along with
+  the analogous uninitialized memory operations.
+- Updated oneDPL use of SYCL to ensure the library is SYCL 2020 compliant.
+
+
+Fixed Issues
+------------
+- Fixed an issue with ``drop_view`` in the experimental range-based API.
+- Fixed compilation errors in ``find_if`` and ``find_if_not`` with device policies where the user provided predicate is
+  device copyable but not trivially copyable.
+- Fixed incorrect results or synchronous SYCL exceptions when the following algorithms are compiled with -O0 and
+  executed on a GPU device: ``copy_if``, ``exclusive_scan``, ``inclusive_scan``, ``partition``, ``partition_copy``,
+  ``reduce_by_segment``, ``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if``, ``sort``, ``stable_partition``,
+  ``transform_exclusive_scan``, ``transform_inclusive_scan``, ``unique``, and ``unique_copy``.
+- Fixed an issue preventing inclusion of the ``<numeric>`` header after ``<execution>`` and ``<algorithm>`` headers.
+- Fixed an issue with range-based ``sort`` and ``stable_sort`` algorithms on Windows
+- Fixed an issue in the ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms
+  that prevented the use of non-trivially-copyable comparators.
+- Fixed kernel name duplication issues in the ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key``
+  algorithms.
+- Fixed the ``sort``, ``stable_sort``, ``sort_by_key``, and ``stable_sort_by_key`` algorithms on devices with sub-group
+  sizes smaller than 4.
+- Fixed synchronization issues in the ``sort``, ``stable_sort``, ``sort_by_key``, ``stable_sort_by_key`` and
+  ``histogram`` algorithms. These issues were known to affect Intel Arc B-Series GPUs.
+- Fixed segmentation faults and incorrect results in ``sort``, ``stable_sort``, ``sort_by_key``, ``stable_sort_by_key``,
+  and ``partial_sort_copy`` when used with a device policy on a CPU device and compiled with -O0 -g.
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- Incorrect results may be observed when calling ``sort`` with a device policy on Intel® Arc™ Graphics 140V integrated
+  graphics with a data sizes of 4-8 million elements.
```

Comment:
```suggestion
  graphics with data sizes of 4-8 million elements.
```

Reply:
I'm not sure what the best way to phrase this issue is. My guess is there is some race condition where the probability of it being exposed increases with input size. Maybe a more generic "with data sizes in the millions of elements" could be an alternative wording.

Reply:
In this case I think calling out the data range where we've observed the issue is helpful.